### ISSUE TITLE
Include build instructions in README so contributors can (easily) get a local Jekyll setup going

### ISSUE THREAD

**Issue by mgudapak** [biopython/biopython.github.io#79]
I can work on this as I am trying to contribute to this repo and need a local Jekyll setup to test my changes anyways ..

**Comment by peterjc** [comment]
Yes please - I'd mentioned a few bits of this on the mailing list, e.g.

I think people should fork the repository but should use their own account name for the repository name - i.e. username.github.io in order to get it rendered by GitHub, e.g. my username is peterjc:

https://github.com/peterjc/peterjc.github.io --> http://peterjc.github.io/

And then we'd need instructions on using Jekyll 3.0, see also https://github.com/biopython/biopython.github.io/blob/master/Gemfile

**Comment by mgudapak** [comment]
i am thinking they (a contributor) should be able to test it locally (as in run jekyll serve on their local machine) and test all pages/links rather than pushing it to their local repo (the fork) and then having GitHub render it under their account. The latter is another way but wouldn't you agree that ideally the contributor should be able to test it even before he pushes it to GitHub (albeit his local/forked repo).

**Comment by peterjc** [comment]
Yes please.

**Comment by LalitNM** [comment]
In my opinion something simple like `CONTRIBUTING.rst` should work here. An example is [here](https://github.com/STAC-IITMandi/Blog/blob/main/CONTRIBUTING.rst).

Advantages of doing this:

- GitHub will sync this file to every issue and pull request page.
- In future if our readme grows, then it would be easy to separate that content in two files.
- This is more common on GitHub.

**Comment by peterjc** [comment]
Yes, but the content needs to be written...

**Comment by LalitNM** [comment]
Sure. I will do that.
